ramp-frontend/ramp_frontend/utils.py

```diff
@@ -55,10 +67,24 @@ def send_mail(to, subject, body):
     body : str
         The body of the email.
     """
+    app = current_app._get_current_object()
     try:
```

Comment:
I am unsure that having a `try/except` in to make async operation is a good idea.
@rth do you know if it is a blocking operation.

Reply:
Yeah, it's not. So here you would just exit the clause and the error will never be handled. I think you would need to also add a try: except clause inside _send_async_email. to log a possible error. logging should be thread safe https://docs.python.org/3/howto/logging-cookbook.html#logging-from-multiple-threads